### DISCUSSION TITLE
Where can i download zygisk lsposed?

### DISCUSSION THREAD

**Discussion by SynapseScribe** [LSPosed/LSPosed#1568] [Q&A] [answered]
Where can i download zygisk lsposed needed for magisk v24 stable?

**Comment by yujincheng08** [comment, marked as answer]
maigsk 24 is not yet stable.

**Reply by SynapseScribe** [reply]
I see it was released yesterday as stable on https://github.com/topjohnwu/Magisk/releases
Am I missing something?

**Reply by SynapseScribe** [reply]
Also, are these zygisk lsposed zips legit from https://forum.xda-developers.com/t/lsposed-xposed-framework-8-0-12-0-simple-magisk-module-edxposed-alternative.4228973/page-31 ?

**Reply by yujincheng08** [reply]
> I see it was released yesterday as stable on https://github.com/topjohnwu/Magisk/releases Am I missing something?

Just beta channel. Not stable channel.

**Reply by yujincheng08** [reply]
> Also, are these zygisk lsposed zips legit from https://forum.xda-developers.com/t/lsposed-xposed-framework-8-0-12-0-simple-magisk-module-edxposed-alternative.4228973/page-31 ?

We don't have any official support in XDA.

**Reply by SynapseScribe** [reply]
uh ok. thanks for the clarifications. How do you know it is magisk beta channel however?

**Reply by yujincheng08** [reply]
https://github.com/topjohnwu/magisk-files

**Reply by SynapseScribe** [reply]
i see. thank you!! good job with the work you do!

**Reply by atimofeev86** [reply]
magisk 24.1 stable now

**Reply by VD171** [reply]
**LSPosed** zygisk stable will be release at 2/2 together **Shamiko** ❤️

**Reply by mywalkb** [reply]
@VD171 Shamiko will be open source or is open source? I can't find nothing, there is only a telegram channel.

**Reply by VD171** [reply]
> @VD171 Shamiko will be open source or is open source? I can't find nothing, there is only a telegram channel.

Just wait and you will see.
